src/components/Admin/Settings/Mail/index.js

```diff
@@ -0,0 +1,421 @@
+            >
+              <FormControlLabel
+                value="Disable"
+                control={<Radio />}
+                label="Disable"
+              />
+              <FormControlLabel value="SMTP" control={<Radio />} label="SMTP" />
+              <FormControlLabel
+                value="Sendgrid"
+                control={<Radio />}
+                label={
+                  <span>
+                    Sendgrid API (See{' '}
+                    <a
+                      href="https://sendgrid.com/docs/ui/account-and-settings/api-keys/#creating-an-api-key"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      here
+                    </a>{' '}
+                    on how to get this token)
+                  </span>
+                }
+              />
+            </RadioGroup>
+            <FlexContainer>
+              <TextField
+                label="Frontend Url"
+                placeholder="Frontend Url"
+                margin="normal"
+                value={frontendUrl || ''}
+                name="frontendUrl"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '36%' }}
+              />
+              <TextField
+                label="Sendgrid Token"
+                placeholder="Sendgrid Token"
+                margin="normal"
+                value={sendgridToken || ''}
+                fullWidth={true}
+                name="sendgridToken"
+                onChange={this.handleChange}
+                disabled={type !== 'Sendgrid'}
+                variant="outlined"
+                style={{ width: '63%' }}
+              />
+            </FlexContainer>
+            <FlexContainer>
+              <TextField
+                label="Trust Self-Signed Certificate"
+                placeholder="Trust Self-Signed Certificate"
+                margin="normal"
+                value={trustselfsignedcerts || ''}
+                name="trustselfsignedcerts"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '32%' }}
+              />
+              <TextField
+                label="Encryption"
+                placeholder="Encryption"
+                margin="normal"
+                value={encryption || ''}
+                name="encryption"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '32%' }}
+              />
+              <TextField
+                label="SMTP Port"
+                placeholder="SMTP Port"
+                margin="normal"
+                value={port || ''}
+                name="port"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '32%' }}
+              />
+            </FlexContainer>
+            <FlexContainer>
+              <TextField
```

Comment:
What is Username and password used for?

Reply:
No idea about it

Reply:
It's present in eventyay admin dashboard, asked a developer